### PR TITLE
New careers lesson: Working and Collaborating Remotely

### PR DESCRIPTION
**Pull request by datadaveshin**
For Hackathon Issue #1295

### CODE REVIEW

File: career/working-and-collaborating-remotely.md [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+- 20 min to Read over Lesson
+- 20 min for Group Practice
+- 10 min for Independent Practice
+- 15 min for Check for Understanding
+
+### Prerequisites
+
+Here are some topics that should be reviewed before this topic:
+- [General Professionalism](professionalism.md)
+- [Workspace Ergonomics](../ergonomics/ergonomics.md)
```
Comment (alodahl) [inline career/working-and-collaborating-remotely.md:16]: we just moved this to a onboarding/ergonomics.md, sorry. 
```suggestion
- [Workspace Ergonomics](../onboarding/ergonomics.md)
```

**Comment by datadaveshin** [comment]
Hi Alina, thanks so much for the review - I'll ask Bill about the Challenges, then update!

**Comment by vegetabill** [comment]
@datadaveshin thanks for working on this. +1 to the challenge being too involved for this stage. I'd say something smaller like having a Zoom to collaboratively do a code challenge would be a better start. That way they could take turns sharing screens and maybe use CodePen or something. What do you think?

**Comment by datadaveshin** [comment]
Yeah, making something really simple using a combo of zoom + trello + codepen would work - I'll refactor and see what you think

**Comment by datadaveshin** [comment]
Whoops - looks like when I accepted Alina's change `[Workspace Ergonomics](../ergonomics/ergonomics.md)` via the gui, I did something that made the branches diverge.. since this is just a single addition of a file, I can just reclone techtonica/curriculum and just submit another PR with the updated file.

**Pull request closed without merging**